AnalyticsRallyHere/Source/AnalyticsRallyHere/Public/RH_AnalyticsProvider.h

```diff
@@ -20,53 +21,74 @@
  * @brief Implementation of analytics for RallyHere GETS.
  */
 class ANALYTICSRALLYHERE_API FRH_AnalyticsProvider :
-	public IAnalyticsProvider,
+	public IAnalyticsProviderET,
 	public FTSTickerObjectBase,
 	public TSharedFromThis<FRH_AnalyticsProvider>
 {
 public:
-	FRH_AnalyticsProvider(const FRH_Analytics::Config& ConfigValues);
+	FRH_AnalyticsProvider(const FAnalyticsET::Config& ConfigValues);
```

Comment:
I'm surprised we wouldn't want to use our own config object for this

Reply:
The interface requires implementing an FAnalyticsET::Config getter and uses that type for some non-virtual functions.  Since our config was effectively a subset of theirs, I just converted over to their type.  We could extend it if we need, in theory, but we would have to be careful about type safety,